src/signals/incident-management/containers/Dashboard/styled.ts

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+
+import {Column, themeSpacing} from "@amsterdam/asc-ui";
+import styled from "styled-components";
+
+export const StyledColumn = styled(Column)`
+  height: 100%;
```

Comment:
we should look for a way to automatically order css properties. would be really convinient.

Reply:
Would agree with you, but I think we should always put css in a different file.